ge/org.osate.ge.fx/src/org/osate/ge/fx/palette/PaletteGroup.java

```diff
@@ -0,0 +1,89 @@
+package org.osate.ge.fx.palette;
+
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.ToggleButton;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.scene.text.Font;
+
+/*
+ * Represents a single group in the palette.
+ *
+ */
+class PaletteGroup<G, I> extends Region {
+	final VBox buttonBox = new VBox();
```

Comment:
All these fields should be private.

Reply:
I have made all the above fields private.